src/app/shared/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1 +1,3 @@
-<p>breadcrumbs works!</p>
+<nav class="breadcrumbs-container">
```

Comment:
please, take a look at https://www.npmjs.com/package/ngx-breadcrumbs npm package. 
Currently it can help to understand better how it can be implemented and then you will be able to use it instead of your custom implementation.

Reply:
Okay, I will see, thanks